app/views/v2/home.html.erb

```diff
@@ -26,7 +26,7 @@
     </header>
     <p>
       Amplify delivers digitised audio files from the collections of cultural,
-      educational and research institutions within Australia.
+      educational and research institutions within Australia.
```

Comment:
I might need glasses, couldn't find any difference 😆

Reply:
![image](https://user-images.githubusercontent.com/86755/42609894-377ac08c-85d2-11e8-9c57-44f2d98b0a65.png)

Oh... ok :)

Reply:
Heh yeah I must have copied and pasted it in from the illustrator file, took a while to find and delete it as it wasn't showing on my screen, only the clients @lucascaton